### ISSUE TITLE
Matroska muxer/demuxer

### ISSUE THREAD

**Issue by brion** [brion/ogv.js#192]
For possible .mkv with h264+opus files...

**Comment by albabosh** [comment]
And a lot of really existing .mkv with h264/ac3 files.

**Comment by Degenerate76** [comment]
FFmpeg has successfully been built with emscripten here: https://github.com/Kagami/ffmpeg.js

If you incorporated libavformat and libavcodec from FFmpeg into this, you would have the ultimate media polyfill, able to play anything in the browser.

would also answer https://github.com/brion/ogv.js/issues/191 and https://github.com/brion/ogv.js/issues/323

**Comment by brion** [comment]
Yes, could probably rig something very general up based on libavformat/libavcodec ... I'm reluctant to distribute a full build myself from within the US (patents yay!) but I definitely want the plugin interface ready so if such a build is made it can be attached to the player logic!
